AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m

```diff
@@ -117,6 +120,19 @@ - (void)setUp {
   self.distributeInfoTrackerMock = OCMClassMock([MSDistributeInfoTracker class]);
   self.sut.distributeInfoTracker = self.distributeInfoTrackerMock;
 
+  // Mock reachability.
+  self.reachabilityMock = OCMClassMock([MS_Reachability class]);
```

Comment:
I have the feeling that we should create a test utility class dedicated to reachability. The current network status would be a property of that class.

Reply:
But it can be a separate work item.

Reply:
I agree that having a separate work item makes sense.